### ISSUE TITLE
4.8.0: no login form on IE11 

### ISSUE THREAD

**Issue by bolera** [phpmyadmin/phpmyadmin#14182]
### Steps to reproduce
1. install 4.8.0 as normal and open the website
2. you get the header/logo and nothing else, no language choice and no login form
3. it seems this is a javascript problem as the console shows an ever reapeating "PMA_ajaxShowMessage is undefined" in error_report.js (29,17). This error repeats very quickly, I have a 1.000 messages now after maybe one, two or three minutes of having the page open. It drives the browser to more than 50% cpu utilization and keeps gulping memory (in tiny steps, so it would take some time to be harmful ...).
4. probably new to 4.8.0. Last good version I tried was 4.7.4, but I have seen reports about other bugs with IE11 for 4.7.9, so this was probably good as well.

### Server configuration
**PHP version:**
7.2
**phpMyAdmin version:**
4.8.0
### Client configuration
**Browser:**
IE 11 latest on Windows 10

**Comment by mauriciofauth** [comment]
This is weird.
This has already been reported in #14061, fixed in 0c2132180366569c9246b45a6ab82642b3119716 and included in 4.8.0.
I could not reproduce this problem with 4.8.0.

**Comment by nulll-pointer** [comment]
@bolera, can you post a screen shot??

**Comment by ibennetch** [comment]
I'm unable to reproduce this with IE11 on Windows 10.

What happens if you access https://demo.phpmyadmin.net/QA_4_8 (username is root and the password is blank, if you get that far).

Was this a new installation of phpMyAdmin or did you upgrade an existing installation?

**Comment by bolera** [comment]
Hm, 4.8.0.1 works. I compared the function.js files. The one I have does not have the fixes from 0c21321 . It's also older than March 15. My file (all files) is dated Feb 27 13:43:40. Oh, my .bash_history says it was 4.8.0-alpha1 and according to the folder I moved it for archiving I may have downloaded it on March 15. I had renamed/shortened the directory I unzipped it to, so I thought it's 4.8.0. So, this was false alarm, sorry. I just don't understand why I would download an alpha version. The 4.7.4 I had was working just fine and I'm really not eager to install even every stable dot release. I close this now.
